app/scenes/CollectionEdit.tsx

```diff
@@ -40,14 +40,17 @@ const CollectionEdit = ({ collectionId, onSubmit }: Props) => {
       setIsSaving(true);
 
       try {
+        const redirect = location.pathname === collection.url;
         await collection.save({
           name,
           icon,
           color,
           sort,
         });
-        history.push(collection.url);
         onSubmit();
+        if (redirect) {
```

Comment:
I'm pretty sure this can be entirely removed as there is already a `useEffect` in the CollectionScene to redirect to the canonical url if it changes